src/exchangeRates.js

```diff
@@ -45,31 +45,34 @@ async function get(params) {
   const today= new Date().toISOString().split('T')[0];
   const {fromCurrency='AUD', toCurrency='USD', onDate=today} = params;
   let exchangeRates = await db.query(
-    `SELECT rate, created_at FROM exchange_rates WHERE from_currency = ? AND to_currency = ? AND on_date = ?`, 
+    `SELECT rate, created_at FROM exchange_rates WHERE from_currency = ? AND to_currency = ? AND on_date = ?`,
     [fromCurrency, toCurrency, onDate]
   );
   if (exchangeRates.length) {
     const rate = Number(exchangeRates[0].rate);
     console.log(`Found exchange rate of ${rate} for ${fromCurrency} to ${toCurrency} of ${onDate} in the db`);
-    
+
     return {fromCurrency, toCurrency, onDate, rate};
   }
-  
+
   return getExternal(fromCurrency, toCurrency, onDate);
 }
 
-async function getMultiple() {
+async function getMultiple(currentPage) {
+
+  let offset = (currentPage - 1) * [config.itemsPerPage];
+
   let allExchangeRates = await db.query(
-    `SELECT from_currency, to_currency, rate, created_at FROM exchange_rates`,
-  );
-  if (allExchangeRates.length) {
+      `SELECT from_currency, to_currency, rate, created_at FROM exchange_rates LIMIT ${offset}, ${config.itemsPerPage}`);
```

Comment:
Can you please select on_date field and use a parameterized query for this with `?` for offset and limit.